### ISSUE TITLE
include x-amz-server-side-encryption AWS S3

### ISSUE THREAD

**Issue by duckyhh** [duplicati/duplicati#2193]
I have:
- [x] searched open and closed issues for duplicates

----------------------------------------

### Version info

**Duplicati Version:** 2.0.1.35
**Operating System:*Windows 7 SP 1x64*
**Backend:**

### Bug description
I want to force AWS S3 Server - Side - Encryption for Backups.
I can't find the option in the expert area from duplicati, so it would be very cool if that option could be implemented.

**https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html**
It's my first feature request, so sorry if something is wrong.

**Comment by BinaryEvolved** [comment]
I agree that this would be a nice feature to add. Although server-side encryption doesn't really protect against much, it is still nice to enable the feature for the few unlikely situations.
